README.md

```diff
@@ -16,7 +16,7 @@ Not familiar with Mux? Check out https://mux.com/ for more information.
 ## Installation
 
 ```
-go get github.com/muxinc/mux-go
+go get github.com/muxinc/mux-go/v4
```

Comment:
hmmm do you expect this to work?

```
$ go get github.com/muxinc/mux-go/v4
go get: module github.com/muxinc/mux-go@upgrade found (v1.1.1), but does not contain package github.com/muxinc/mux-go/v4```

Reply:
No, not until this PR is merged sadly as this PR also bumps the declaration in `go.mod`

Reply:
yea k that makes sense to me. ship it.